### ISSUE TITLE
[DOC-FIX] Syntax error on mlflow.org website on `load_model`, correct on Github docs

### ISSUE THREAD

**Issue by franckjay** [mlflow/mlflow#3612]
Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md) for information on what types of issues we address.

**Please fill in this documentation issue template to ensure a timely and thorough response.**

### Willingness to contribute
The MLflow Community encourages documentation fix contributions. Would you or another member of your organization be willing to contribute a fix for this documentation issue to the MLflow code base?

- [ ] Yes. I can contribute a documentation fix independently.
- [x ] Yes. I would be willing to contribute a document fix with guidance from the MLflow community.
- [ ] No. I cannot contribute a documentation fix at this time.

### URL(s) with the issue:
https://mlflow.org/docs/latest/model-registry.html#fetching-an-mlflow-model-from-the-model-registry 
https://github.com/mlflow/mlflow/blob/master/docs/source/model-registry.rst#fetching-an-mlflow-model-from-the-model-registry
### Description of proposal (what needs changing):

On the HTML page, the `load_model` command is missing a closing `"`, but in the Github docs it is correctly displayed:
```
model = mlflow.pyfunc.load_model(
    model_uri=f"models:/{model_name}/{model_version}"
)
```

**Comment by dmatrix** [comment]
@franckjay thanks for catching it. It's a quick fix. Either you can file a PR or I can fix it since I'm working on a couple of PRs. Let me know.

**Comment by dmatrix** [comment]
This is already fixed in #3504

![Screen Shot 2020-10-30 at 6 00 57 PM](https://user-images.githubusercontent.com/1117597/97767552-fe345500-1ad9-11eb-9092-238f86669303.png)
